client/settings/general-settings-section/index.js

```diff
@@ -2,52 +2,85 @@
  * External dependencies
  */
 import React from 'react';
+import styled from '@emotion/styled';
 import { Card } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import CardBody from '../card-body';
-import SofortIcon from '../../payment-method-icons/sofort';
-import SepaIcon from '../../payment-method-icons/sepa';
 import CardsIcon from '../../payment-method-icons/cards';
-import GiropayIcon from '../../payment-method-icons/giropay';
-import ApplePayIcon from '../../payment-method-icons/apple-pay';
-import GooglePayIcon from '../../payment-method-icons/google-pay';
+import UPEOptInBanner from '../upe-opt-in-banner';
+
+const GeneralSettingsSectionWrapper = styled.div`
+	display: flex;
+	flex-direction: column;
+`;
+
+const CardBodyWrapper = styled( CardBody )`
+	display: flex;
+
+	> * {
+		margin-bottom: 0px;
+	}
+`;
+
+const PaymentMethodText = styled.div`
+	flex: 0 0 100%;
+
+	@media ( min-width: 660px ) {
+		flex: 1 1 auto;
+		margin-left: 12px;
+	}
+`;
+
+const PaymentMethodLabel = styled.div`
+	color: $gray-900;
+	display: inline-block;
+	font-size: 14px;
+	font-weight: 600;
+	line-height: 20px;
+	margin-bottom: 4px;
+`;
+
+const PaymentMethodDescription = styled.div`
+	color: ##646970;
+	font-size: 13px;
+	line-height: 16px;
+	margin-bottom: 14px;
+
+	@media ( min-width: 660px ) {
+		margin-bottom: 0px;
+	}
+`;
+
+const OptInBannerWrapper = styled.div`
+	div:first-of-type {
+		max-width: 100%;
+	}
+`;
 
 const GeneralSettingsSection = () => {
 	return (
-		<Card>
-			<CardBody>
-				The general settings sections goes here.
-				<ul>
-					<li>
-						<GooglePayIcon />
-						<GooglePayIcon size="medium" />
-					</li>
-					<li>
-						<ApplePayIcon />
-						<ApplePayIcon size="medium" />
-					</li>
-					<li>
-						<SofortIcon />
-						<SofortIcon size="medium" />
-					</li>
-					<li>
-						<GiropayIcon />
-						<GiropayIcon size="medium" />
-					</li>
-					<li>
-						<SepaIcon />
-						<SepaIcon size="medium" />
-					</li>
-					<li>
-						<CardsIcon />
-						<CardsIcon size="medium" />
-					</li>
-				</ul>
-			</CardBody>
-		</Card>
+		<GeneralSettingsSectionWrapper>
+			<Card>
+				<CardBodyWrapper>
+					<CardsIcon size="medium" />
+					<PaymentMethodText>
+						<PaymentMethodLabel>
+							Credit card / debit card
+						</PaymentMethodLabel>
+						<PaymentMethodDescription>
+							Let your customers pay with major credit and debit
+							cards without leaving your store.
+						</PaymentMethodDescription>
+					</PaymentMethodText>
+				</CardBodyWrapper>
+			</Card>
+			<OptInBannerWrapper data-testid="opt-in-banner">
```

Comment:
Let's add "UPE" to the name to make it a bit clearer what banner this is:

```suggestion
			<UPEOptInBannerWrapper data-testid="upe-opt-in-banner">
```